public_html/index.html

```diff
@@ -1,151 +1,146 @@
-with clients offering a unique expertise that helps them achieve their
-objectives.</p>
-
-<h2 id="toc_11">What makes C Baily Law LLC unique</h2>
-
-<p>Here&rsquo;s what people are saying about Christine Baily and her legal
-expertise.</p>
-
-<h3 id="toc_12">Testimonials</h3>
-
-<p><q>I highly recommend Christine to clients seeking expert guidance
-navigating complex regulatory situations. I have worked closely with
-her, and she brings tenacious advocacy and deep industry experience to
-achieve practical solutions.</q></p>
-
-<blockquote>
-<p>&mdash; A state government lawyer.</p>
-</blockquote>
-
-<p><q>Christine has a remarkable aptitude of tackling the issues brought to
-her by her clients with humility, keen sensitivity, and, invariably,
-from an angle not previously contemplated by the client.</q></p>
-
-<blockquote>
-<p>&mdash; A federal government lawyer.</p>
-</blockquote>
-
-<h3 id="toc_13">In the press</h3>
-
-<p>Commonwealth Magazine reported on an amicus brief submitted in the
-matter <em>Mederi, Inc. v. City of Salem</em>, a
-case before the Massachusetts Supreme Judicial Court:</p>
-
-<blockquote>
-<p><q>Christine Baily, an attorney for the Cannabis Control Commission &hellip;
-notes the inconsistency in the law. Baily writes that by granting
-municipalities exclusive control over the host community agreement
-process, municipalities have authority over who can seek state
-licensure.</q></p>
-
-<p>&mdash; <a href="https://commonwealthmagazine.org/marijuana/sjc-takes-up-local-control-of-marijuana-licensing/.">SJC takes up local control of marijuana
-licensing</a></p>
-</blockquote>
-
-<p>When the SJC issued its decision, <em>Mederi, Inc. v. City of Salem</em>, 488 Mass. 60 (2021), the Boston Bar
-Association noted that the court raised <q>valuable questions about the
-purposes of HCAs.</q> <a href="https://bostonbar.org/journal/massachusetts-high-courts-weigh-in-on-the-limits-of-local-control-over-cannabis-businesses/">Massachusetts High Courts Weigh In On the Limits of
-Local Control Over Cannabis Businesses - Boston Bar
-Association</a>.
-These questions reflected its close consideration of the points made in
-her brief.</p>
-
-<h2 id="toc_14">Contact Us</h2>
-
-<p>Please let me know if you are seeking a free evaluation of whether the
-firm can represent you in a legal matter.</p>
-
-<p>11 Adams Street<br />
-Arlington, MA 02474</p>
-
-<p>(617) 610-9798</p>
-
-<p><strong>Insert Clio contact form here.</strong></p>
-
-<h2 id="toc_15">Footer</h2>
-
-<p>Thank you for visiting the website of C Baily Law LLC. Information
-offered on the website is intended for informational purposes only and
-should not be considered legal advice.</p>
+    <!-- Start header section -->
+    <header id="cbailylaw_header">
+        <div class="cbailylaw_header_inner">
+            <!-- logo -->
+            <div class="cbailylaw__logo">
+                <a href="https://www.cbailylaw.com/">C. Baily Law</a>
```

Comment:
To ensure these links will work on the staging server, which will have a different domain name, please use the following format for the current directory, regardless of domain:

```
<a href="./">home</a>
```